src/test/java/seedu/address/logic/commands/DeleteStudentCommandTest.java

```diff
@@ -50,7 +50,7 @@ public void execute_studentDelete_success() throws Exception {
 
         CommandResult result = command.execute(modelStub);
 
-        assertEquals(String.format(DeleteStudentCommand.MESSAGE_DELETE_STUDENT_SUCCESS, validStudent.getName()),
+        assertEquals(String.format(DeleteStudentCommand.MESSAGE_DELETE_STUDENT_SUCCESS + validStudent.getName()),
```

Comment:
Btw what wee kean meant was for this. So how String format works is if i have a format flag like "$1%s" it means I will substitute that flag with the first string argument you provide.
![image](https://github.com/user-attachments/assets/dfd65c4c-9cdf-4d32-8ff8-032b2654f98b)
Here's the code if you still want to use String format.